src/Nox.Generator/Application/IntegrationEvents/DefaultIntegrationEvent.template.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Generated
+
+#nullable enable
+
+using Nox.Abstractions;
+using Nox.Application;
+
+using {{codeGeneratorState.ApplicationNameSpace}}.Dto;
+
+namespace {{codeGeneratorState.ApplicationNameSpace}}.IntegrationEvents;
+
+/// <summary>
+/// {{entity.Name}}{{crudOperation}} integration event.
+/// </summary>
+internal record {{entity.Name}}{{crudOperation}}({{entity.Name}}Dto {{entity.Name}}) :  IIntegrationEvent;
```

Comment:
just a suggestion are y ok in renaming in this PR crudOperation to operation?

Reply:
Changes were applied.